account_product_fiscal_classification/models/wizard_change_fiscal_classification.py

```diff
@@ -27,11 +27,14 @@ def _default_old_fiscal_classification_id(self):
         required=True, domain="[('id', '!=', old_fiscal_classification_id)]")
 
     # View Section
-    @api.one
```

Comment:
From what I can understand, it is deprecated but not discontinued. Source https://github.com/odoo/odoo/issues/12408#issuecomment-226421501 . My purpose of removal was lint check https://goo.gl/CVVnks